collectors/nvd/collectors.py

```diff
@@ -56,32 +58,72 @@ def response2result(self, vulnerabilities: list) -> list:
         filtering out everything unnecessary and simplifying
         """
 
-        def compose_cvss(metrics: list) -> Union[str, None]:
+        def get_description(descriptions: list) -> Union[str, None]:
             """
-            compose the result vector helper
+            Return a description only if it is written in English.
             """
-            for metric in metrics:
-                # we only care for NVD record
-                if metric.source == "nvd@nist.gov":
-                    return f"{metric.cvssData.baseScore}/{metric.cvssData.vectorString}"
+            return [d.value for d in descriptions if d.lang == "en"][0] or None
 
-        def get_metric(metrics: dict, version: str) -> Union[str, None]:
+        def get_references(data: CVE) -> dict:
             """
-            get metric record for the given version if exists
+            Return the source URL and all other URLs.
             """
-            if version not in metrics:
+            urls = [{"SOURCE": f"https://nvd.nist.gov/vuln/detail/{data.id}"}]
```

Comment:
The dictionary does not correspond to `FlawReference` model. It should be rather
```
{
    "type": "SOURCE",
    "url": f"https://nvd.nist.gov/vuln/detail/{data.id}"},
}
```

Reply:
Oh, my bad. Thanks :)